l10n_it_delivery_note/wizard/delivery_note_invoice.py

```diff
@@ -37,5 +37,4 @@ def create_invoices(self):
         delivery_note_ids.action_invoice(self.invoice_method)
         for invoice in delivery_note_ids.mapped("invoice_ids"):
             invoice.invoice_date = self.invoice_date
-            invoice._onchange_invoice_date()
```

Comment:
**question**: possibile che la funzione abbia cambiato nome in v16 e sia `_onchange_date()`?

v14: https://github.com/odoo/odoo/blob/14.0/addons/account/models/account_move.py#L414
v16: https://github.com/Odoo/odoo/blob/16.0/addons/account/models/account_move.py#L1586